src/Windowing/OpenToolkit.GraphicsLibraryFramework/GLFW.cs

```diff
@@ -387,5 +387,20 @@ protected GLFW(string path, ImplementationOptions options)
 
         /// <inheritdoc />
         public abstract unsafe void SetClipboardString(Window* window, string data);
+
+        /// <inheritdoc />
+        public abstract bool VulkanSupported();
+
+        /// <inheritdoc />
+        public abstract unsafe byte** GetRequiredInstanceExtensions(out uint count);
+
+        /// <inheritdoc />
+        public abstract unsafe IntPtr GetInstanceProcAddress(IntPtr instance, byte* procName);
```

Comment:
could you use a empty stuct as an opaque handle and pointers please, instead of IntPtr?

Reply:
The Vulkan handles have to be the actual handles themselves, not pointers to them. I figured using `IntPtr` was for the best since Vulkan implementations do expose the handle value as `IntPtr` too.

Also, should I make the return value a pointer type?

Reply:
well then any struct containing an IntPtr would be still be better. So that we have some kind of type safety. And performance wise it's the same as both needs to be blitted.

well the return value IntPtr is fine I guess as with calli IntPtr should be fine...